storage/lmdb/tests/backend.rs

```diff
@@ -16,7 +16,7 @@
 use storage_lmdb::Lmdb;
 
 fn main() {
-    let test_root = test_utils::test_root!("backend-tests").unwrap();
+    let test_root = test_utils::test_root!("backend-tests/lmdb").unwrap();
```

Comment:
I later realised that the `test_root!` macro already incorporates the crate name into the path, so this change is not needed and can be reverted. Same in the sqlite backend. My mistake, apologies.